### PR TITLE
Added manifest for OpenAMP virtio projects.

### PR DESCRIPTION
**Pull request by danmilea**
OpenAMP virtio west manifest.

### CODE REVIEW

File: west-virtio-exp.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+manifest:
+  remotes:
+    - name: openamp
+      url-base: https://github.com/OpenAMP
+    - name: zephyr
```
Comment (arnopo) [inline west-virtio-exp.yml:8]: do you  need the Zephyr remote as you point to  openamp-zephyr-staging?

Comment (danmilea) [reply west-virtio-exp.yml]: I removed the zephyr remote.

File: west-virtio-exp.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2022 STMicroelectronics
```
Comment (arnopo) [inline west-virtio-exp.yml:1]: could you update th copyright before I merge it

Comment (danmilea) [reply west-virtio-exp.yml]: Done.

**Review by arnopo — APPROVED** [review]

**Pull request merged**